lib/repl.js

```diff
@@ -270,8 +271,8 @@ function REPLServer(prompt,
 
   if (options.terminal && options.useColors === undefined) {
     // If possible, check if stdout supports colors or not.
-    if (options.output.hasColors) {
-      options.useColors = options.output.hasColors();
+    if (shouldColorize(options.output)) {
+      options.useColors = true;
     } else if (process.env.NODE_DISABLE_COLORS === undefined) {
       options.useColors = true;
     }
```

Comment:
any reason not to do this?:
```suggestion
    options.useColors = shouldColorize(options.output) || process.env.NODE_DISABLE_COLORS === undefined;
```